app/controllers/concerns/rendering.rb

```diff
@@ -11,14 +11,24 @@ def render_json(model, **args)
       render({ json: serializer.new(model, opts) }.merge(args))
     end
 
-    def render_error(model, **args)
+    def render_error(models, **args)
       render({ json: {
-        errors: model.errors.full_messages
+        errors: model_errors(models)
       }, status: :not_acceptable }.merge(args))
     end
 
     private
 
+    def model_errors(models = [])
+      models = [models].flatten
+      models.flat_map do |model|
+        model.errors.full_messages.map do |error|
+          error[0] = error[0].downcase
+          "#{model.class.name} #{error}"
+        end
+      end
+    end
```

Comment:
Since the V1 profiles controller now creates/updates profiles and policies, we need to be able to show errors of both models. This does not break the old functionality of only passing a single model, via `models = [models].flatten`.